scripts/generate/templates/_partials/_component_requirements.md.erb

```diff
@@ -35,6 +35,9 @@
 <%- if requirements.prometheus -%>
 * [Prometheus][urls.prometheus] version `<%= requirements.prometheus %>` is required.
 <%- end -%>
+<%- if requirements.feature -%>
+* Vector must be build with the `<%= requirements.feature %>` feature. *This is not enabled by default. Review [Building Vector](https://github.com/timberio/vector/blob/master/CONTRIBUTING.md#setup) to enable.)*
```

Comment:
```suggestion
* Vector must be build with the `<%= requirements.feature %>` feature. *This is not enabled by default. Review [Building Vector][urls.contributing#setup] to enable.)*
```

This will require adding `contributing` to the `links.toml` file.

Reply:
Ack whoops, thanks!